### PR TITLE
Initial scaffold of Index page with LayoutNav component

### PR DESCRIPTION
**Pull request by diegonvs**
### Reminder:
Hey @matuzalemsteles, don't forget the `` with children issue.

### CODE REVIEW

File: src/components/LayoutNav/LayoutNav.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    getScrollTop() {
+        if (this.node == window) {
+            return this.node.pageYOffset;
+        }
+        if (this.node == document) {
+            return this.node.defaultView.pageYOffset;
+        }
+        return this.node.scrollTop;
```
[the remark this answers is not in the document]
Comment (diegonvs) [reply src/components/LayoutNav/LayoutNav.js]: 🤔

[the remark this answers is not in the document]
Comment (diegonvs) [reply src/components/LayoutNav/LayoutNav.js]: I missed it

**Comment by diegonvs** [comment]
Ye, i'll start rebase right now! XD

File: src/components/LayoutNav/LayoutNav.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+
+class LayoutNav extends Component {
+    constructor() {
+        super();
+        this._rootNode = window || document;
```
Comment (matuzalemsteles) [inline src/components/LayoutNav/LayoutNav.js]: hey @diegonvs, Did you check if there still happens the error of not finding the `window` in the build? I do not see those utils that we had talked about in your desk.

**Pull request merged**